modules/accredited_representative_portal/db/migrate/20250128175216_create_ar_user_account_accredited_individuals.rb

```diff
@@ -0,0 +1,14 @@
+class CreateArUserAccountAccreditedIndividuals < ActiveRecord::Migration[7.2]
+  def change
+    create_table :ar_user_account_accredited_individuals, id: :uuid do |t|
+      t.string :accredited_individual_registration_number, null: false
+      t.string :power_of_attorney_holder_type, null: false
+      t.string :user_account_email, null: false
+      t.string :user_account_icn
+    end
+
+    add_index :ar_user_account_accredited_individuals,
+      :accredited_individual_registration_number,
+      unique: true
+  end
```

Comment:
Will platform ask for?
- `disable_ddl_transaction!`
- `algorithm: :concurrently`

Reply:
yes 😄 
And please include this index update in a separate migration.

Reply:
@ojbucao [here are the docs](https://depo-platform-documentation.scrollhelp.site/developer-docs/vets-api-database-migrations#vets-apidatabasemigrations-Indexes) explaining that.